app/Transformers/BeatmapPackTransformer.php

```diff
@@ -26,11 +26,10 @@ public function transform(BeatmapPack $pack)
         return [
             'author' => $pack->author,
             'date' => $pack->date,
-            'hidden' => $pack->hidden,
             'name' => $pack->name,
             'no_diff_reduction' => $pack->no_diff_reduction,
             'pack_id' => $pack->pack_id,
-            'ruleset' => $pack->getRulesetAttribute(),
+            'ruleset_id' => $pack->getRulesetIdAttribute(),
```

Comment:
just use `$this->playmode`; also, `getThingAttribute()`s can just be accessed by `->thing`